packages/cli/src/lib/graphql/admin/link-storefront.ts

```diff
@@ -14,6 +17,23 @@ interface HydrogenStorefront {
   productionUrl: string;
 }
 
-export interface LinkStorefrontSchema {
+interface LinkStorefrontSchema {
   hydrogenStorefronts: HydrogenStorefront[];
 }
+
+export async function getStorefronts(shop: string) {
+  const adminSession = await getAdminSession(shop);
+
+  const {hydrogenStorefronts} = await adminRequest<LinkStorefrontSchema>(
+    LinkStorefrontQuery,
+    adminSession,
+  );
+
+  return {
+    adminSession,
+    storefronts: hydrogenStorefronts.map((storefront) => ({
```

Comment:
Well since this is always "hydrogen", I thought "hydrogenStorefronts" was just redundant and we could rely on types. But perhaps a longer name is still better 🤔

Reply:
Feel free to keep it! I think we've got some storefront/shop/store renames coming anyways.